lib/instana/instrumentation/aws_sdk_lambda.rb

```diff
@@ -25,19 +25,12 @@ def call(context)
             type: context.params[:invocation_type]
           }.reject { |_, v| v.nil? }
 
-          span = ::Instana.tracer.log_entry(:"aws.lambda.invoke", {aws: {lambda: {invoke: tags}}})
-          begin
+          ::Instana.tracer.start_or_continue_trace(:"aws.lambda.invoke", {aws: {lambda: {invoke: tags}}}) do
             response = @handler.call(context)
             if response.respond_to? :status_code
-              span.set_tags(:http => {:status => response.status_code })
```

Comment:
So, we are not setting the tag into the span data structure, but only logging it? Why?

Reply:
It is set with the `log_info` call. You can check that function here https://github.com/instana/ruby-sensor/blob/master/lib/instana/tracer.rb#L151. It's name is quite misleading indeed.

Reply:
Got it.